src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1011"
+  },
+  "description": {
+    "text": "An empty project template for creating an ASP.NET Core application. This template does not have any content in it.",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1012"
+  },
+  "order": 100,
+  "icon": "vs-2017.3/Empty.png",
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?LinkID=784883",
+  "uiFilters": [ "oneaspnet" ],
+  "supportsDocker": true,
+  "legacyTemplateIdentity": "Microsoft.NetCore.FSharp.EmptyWeb",
+  "ports": [
```

Comment:
These port specifications don't seem correct, but @phenning would know better.

Reply:
This should in general line up with whats in the template.config file, this tells VS which ports to generate.

@javiercn  while looking at this, I have some concerns that you made to the host file . We need to discuss this offline.

Reply:
Aligning it to be like the C# template for now. @javiercn let me know once you've made your change so I can add the HttpPort as well.